sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -186,3 +191,143 @@ public partial class TableResponseProperties
         public string TableName { get { throw null; } }
     }
 }
+namespace Azure.Data.Tables.Sas
+{
+    public partial class AccountSasBuilder
+    {
+        public AccountSasBuilder() { }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } set { } }
+        public Azure.Data.Tables.Sas.SasIPRange IPRange { get { throw null; } set { } }
+        public string Permissions { get { throw null; } }
+        public Azure.Data.Tables.Sas.SasProtocol Protocol { get { throw null; } set { } }
+        public Azure.Data.Tables.Sas.AccountSasResourceTypes ResourceTypes { get { throw null; } set { } }
+        public Azure.Data.Tables.Sas.AccountSasServices Services { get { throw null; } set { } }
+        public System.DateTimeOffset StartsOn { get { throw null; } set { } }
+        public string Version { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public void SetPermissions(Azure.Data.Tables.Sas.AccountSasPermissions permissions) { }
+        public void SetPermissions(string rawPermissions) { }
+        public Azure.Data.Tables.Sas.SasQueryParameters ToSasQueryParameters(Azure.Data.Tables.TablesSharedKeyCredential sharedKeyCredential) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override string ToString() { throw null; }
+    }
+    [System.FlagsAttribute]
+    public enum AccountSasPermissions
+    {
+        All = -1,
+        Read = 1,
+        Write = 2,
+        Delete = 4,
+        List = 8,
+        Add = 16,
+        Create = 32,
+        Update = 64,
+    }
+    [System.FlagsAttribute]
+    public enum AccountSasResourceTypes
+    {
+        All = -1,
+        Service = 1,
+        Container = 2,
+        Object = 4,
+    }
+    [System.FlagsAttribute]
+    public enum AccountSasServices
```

Comment:
Doesn't seem this enum is required.

Reply:
I'll leave it for now pending the discussion about account level sas.